src/Tools/Test Framework/Test Runner/src/CommandLineTestTool.Page.al

```diff
@@ -96,6 +96,25 @@ page 130455 "Command Line Test Tool"
                     if Rec.FindFirst() then;
                 end;
             }
+            field(TestType; TestType)
```

Comment:
This must be an integer. We must avoid being affected by the translations. See how the other types were handled.

Reply:
Decided to make it a `Text` field because it would be more readable in the build scripts.

Not sure if this will be impacted by translation tbh, since the value come from build script, and we convert it to Integer with `ConvertTestTypeToOption`

```
function Set-TestType
(
    [ValidateSet('UnitTest','IntegrationTest','Uncategorized')]
    [string] $TestType,
...
    $testTypeControl = $ClientContext.GetControlByName($Form, "TestType")
}
```

Reply:
Let's follow the same patterns here as other options.